helm/README.md

```diff
@@ -93,7 +88,7 @@ kubectl create namespace inference-ms
 A command like the one below will then use the latest chart version to install the version of NIM defined in the values file into the `inference-ms` namespace in your Kubernetes cluster. Modify it as required.
 
 ```bash
-helm --namespace inference-ms install my-nim nim-demo/nim-llm -f path/to/your/custom-values.yaml
+helm --namespace inference-ms install my-nim nim-llm -f path/to/your/custom-values.yaml
```

Comment:
nit: if you make it `nim-llm/` then it looks more definite that you are using a local directory. Same for helm show values (which honestly doesn't apply as much for a chart you just downloaded anyway).

Reply:
Sure, done